doc/install/install-ceph-gateway.rst

```diff
@@ -510,24 +510,24 @@ Execute the following steps:
 
 #. Add the following contents to the file::
 
-    import boto
```

Comment:
@hansbogert. okay, i trust you. but how come python knows what `boto` is without this statement. weird...